docs/meshstack.kubernetes.metering.md

```diff
@@ -13,4 +13,33 @@ meshStack supports metering and billing for most common Kubernetes versions.
 
 ## Supported Resources
 
-Currently Kubernetes metering is based on Pods. They are metered via the configured limit of CPU and memory of their containers.
+Resources with the following traits are addressable in the [Product Catalog](meshstack.billing-configuration.md#defining-a-custom-product-catalog). Operators can use the traits of these resources to define fine-granular product and pricing rules.
+
+### Pod
+
+<!--snippet:mesh.kraken.productcatalog.traits.kubernetes.pod#type-->
+
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Dhall Type-->
+```dhall
+let PodResourceTraits =
+    {-
+      milliCpu:
+        Configured total CPU limit of all containers in this Pod, in milli CPU (1/1000 of a CPU).
+
+      ramMb:
+        Configured total RAM limit of all containers in this Pod, in MiB.
+
+      status:
+        Status phase of this pod. See https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/
+    -}
+      { milliCpu : Integer, ramMb : Integer, status : Text }
+```
+<!--Example-->
+```dhall
+let example
+    : PodResourceTraits
+    = { milliCpu = +2500, ramMb = +1024, status = "Running" }
```

Comment:
d: is the + really supported here?

Reply:
that's unfortunately a bit of a dhall "oddity", i.e. a literal "2500" would be a `Natural` (floating point) while the "+2500" makes it an  `Integer`

Reply:
ok